src/shared/proto/messages_robocup_ssl_detection_tracked.proto

```diff
@@ -0,0 +1,106 @@
+syntax = "proto2";
+
+message Vector2 {
+    required float x = 1;
+    required float y = 2;
+}
+
+message Vector3 {
+    required float x = 1;
+    required float y = 2;
+    required float z = 3;
+}
+
+// The team color of the robot
+enum TeamColor {
+    UNKNOWN = 1;
+    YELLOW = 2;
+    BLUE = 3;
+}
+
+message RobotId {
+    // The robot number
+    required uint32 id = 1;
+    // The team color
+    required TeamColor team_color = 2;
+}
+
+// A single tracked ball
+message TrackedBall {
+    // The quality of the detection
+    // A value between 0 (not visible) and 1 (visible)
+    // The exact implementation depends on the source software
+    optional float quality = 1;
+
+    // The position (x, y, height) [m] in the ssl-vision coordinate system
+    required Vector3 pos = 2;
+
+    // The velocity [m/s] in the ssl-vision coordinate system
+    optional Vector3 vel = 3;
+}
+
+// A ball kicked by a robot, including predictions when the ball will come to a stop
+message KickedBall {
+    // The initial position [m] from which the ball was kicked
+    required Vector3 pos = 1;
+    // The initial velocity [m/s] with which the ball was kicked
+    required Vector3 vel = 2;
+    // The unix timestamp [s] when the kick was performed
+    required double timestamp = 3;
+
+    // The predicted time [s] that ball ball needs to come to a stop
+    optional float stop_time = 4;
+    // The predicted position [m] at which the ball will come to a stop
+    optional Vector2 stop_pos = 5;
+
+    // The robot that kicked the ball
+    optional RobotId robot_id = 6;
+}
+
+// A single tracked robot
+message TrackedRobot {
+    required RobotId robot_id = 1;
+
+    // The quality of the detection
+    // A value between 0 (not visible) and 1 (visible)
+    // The exact implementation depends on the source software
+    optional float quality = 2;
+
+    // The x-position [m] in the ssl-vision coordinate system
+    required float pos_x = 3;
```

Comment:
Why not use the Vector messages from above here?

Reply:
good catch. I forgot it there after introducing the vector messages. Fixed now.